src/Resources/Resources/help/New-AzADServicePrincipal.md

```diff
@@ -130,6 +130,11 @@ assigned to the service principal. The default values for the **Role** and **Sco
 properties if an ApplicationId is not provided. To update the application-specific parameters, use
 the [Update-AzADApplication](./update-azadapplication.md) cmdlet.
 
+> [!WARNING]
+> When you create a service principal using the **New-AzADServicePrincipal** command, the output includes credentials that you must protect. Be sure that you do not include these credentials in your code or check the credentials into your source control. As an alternative, consider using [managed identities](/azure/active-directory/managed-identities-azure-resources/overview) to avoid the need to use credentials.
```

Comment:
Hi @rolyon , do you think we can use the full docs.microsoft.com URL instead? For example

```suggestion
> When you create a service principal using the **New-AzADServicePrincipal** command, the output includes credentials that you must protect. Be sure that you do not include these credentials in your code or check the credentials into your source control. As an alternative, consider using [managed identities](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview) to avoid the need to use credentials.
```

Because otherwise it would be confusing if customer is reading the help within a terminal (through `Get-Help New-AzADServicePrincipal`)

Reply:
Relative URL's are preferred from a docs standpoint.

Reply:
After more consideration, the full URL should be used in this scenario. Reference docs ship with the module itself. This means that a relative URL will not work when used on the users local system with the `Get-Help` cmdlet.

Reply:
I have confirmed that URL's are removed when converting to MAML so relative URL's are preferred.